### ISSUE TITLE
[Docs]: s3 bucket notification policy issue

### ISSUE THREAD

**Issue by LeniM** [hashicorp/terraform-provider-aws#32373]
### Documentation Link

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket_notification

### Description

Hi all,

The queue policy document in the example is not functional. See below the value vs the correction. 
```
 "arnawssqs:*:*:s3-event-notification-queue" # is the value given in the example
 "arn:aws:sqs:*:*:s3-event-notification-queue" # correct value
```

The example result with the following error results 
```
Error: error putting S3 Bucket Notification Configuration: InvalidArgument: Unable to validate the following destination configurations
 	status code: 400, request id: ...., host id: ....
```

### Would you like to implement a fix?

None

**Comment by divyafun** [comment]
I would like to work on this

**Comment by nicovillanueva** [comment]
There's also an issue with the example event names.

Where it says:
```events        = ["s3ObjectCreated*"]
```

It should be:
```events        = ["s3:ObjectCreated:*"]
```
(note the missing colons)

This applies to all examples on this page.

**Comment by justinretzolk** [comment]
Duplicate of #32378

Hey all 👋 Thank you for taking the time to raise this and participate in this discussion. I happened to catch https://github.com/hashicorp/terraform-provider-aws/issues/32378 before seeing this issue, so will be using https://github.com/hashicorp/terraform-provider-aws/issues/32378 as a point to gather feedback such as this. This appears to be a change on the Registry side, presumably unintentionally. I've raised a ticket with the appropriate team internally, and will close this issue as a duplicate, preferring to update the other ticket.
